packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -506,9 +507,7 @@ function getLargestEntityFromBatch(
  * @param data
  * @returns
  */
-function getLargestRawDataEntryFromEntity(
-  data: EntityRawData[],
-): EntityRawData {
+function getLargestKeyFromData(data: EntityRawData[]): EntityRawData {
```

Comment:
Technically, this helper function is returning the entire item, not just the key, but I otherwise like the name change.